test_bst.py

```diff
@@ -75,6 +77,31 @@ def test_contains(filled_tree):
         assert t.contains(num) is False
 
 
+def test_in_order(filled_tree):
+    tree = filled_tree
+    for place, item in enumerate(tree.in_order()):
+        assert place == item
+
+
+def test_pre_order(filled_tree):
+    tree = filled_tree
+    expected_order = [9, 8, 7, 6, 5, 4, 3, 2, 1, 0, 10, 11, 12, 13]
+    for place, item in enumerate(tree.pre_order()):
+        assert expected_order[place] == item
```

Comment:
Again, if nothing comes out, this fails to prove anything.  If only one item comes out, it still only proves that the right first value appears.  There's nothing in here that proves that the entire expected list appears.